python/tvm/contrib/micro/meta_schedule/test_autotune_ms.py

```diff
@@ -0,0 +1,181 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
You need to move this file outside of the python package. Usually we have CRT test and Zephyr/Arduino tests in separate path. CRT test would be in the python unittests and platform specific tests are in `tests/micro`

Reply:
done.